molgenis-navigator/src/main/java/org/molgenis/navigator/copy/job/CopyJobExecution.java

```diff
@@ -0,0 +1,50 @@
+package org.molgenis.navigator.copy.job;
+
+import static org.molgenis.navigator.copy.job.CopyJobExecutionMetadata.COPY_JOB_TYPE;
+import static org.molgenis.navigator.copy.job.CopyJobExecutionMetadata.RESOURCES;
+import static org.molgenis.navigator.copy.job.CopyJobExecutionMetadata.TARGET_PACKAGE;
+import static org.molgenis.navigator.util.ResourceIdentifierUtil.getResourcesFromJson;
+
+import com.google.gson.Gson;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.molgenis.data.Entity;
+import org.molgenis.data.meta.model.EntityType;
+import org.molgenis.jobs.model.JobExecution;
+import org.molgenis.navigator.model.ResourceIdentifier;
+
+@SuppressWarnings({"unused", "WeakerAccess"})
+public class CopyJobExecution extends JobExecution {
```

Comment:
I can imagine another module also wanting to copy/download something in a job. ```ResourceCopyJobExecution``` might be a better name? Keep in mind that changing entity type identifiers is not feasible once this type is introduced. Same goes for download @bartcharbon.